packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -0,0 +1,118 @@
+
+#### Heading, Paragraph Components
+We'll expand on the Text primitive to build out Heading and Paragraph components.
+
+#### Card Component
+We're designing and building a unified card component for Paste. This will replace our legacy card soup with a composable card that you can customize for your products.
+
+#### Grid Component (Design Preview)
+A standard 12-column grid for Paste will drop in 0.3. There's been a few discussions around responsiveness, and to give some clarity:
+
+- This will be a responsive grid layout system
+- All components will be designed to be 100% width to their containers.
+
+_Note:_ Just like accessibility, responsiveness is not an out-of-the-box solution. Consumers of any design system still need to compose their responsive screens based on their unique requirements. We'll provide the layout system and fluid components. Teamwork!
+
+#### Flex Component
+A layout component that simplifies aligning a single horizontal row of elements. Also has a fondness for protein shakes.
+
+#### Paste Utilities
+You remember the first time you uploaded a profile picture to AIM and it was skewed out? No, that was just us? Well, the Aspect Ratio utility will drop in this release and give guidance on how to maintain proportions while embedding images, video, media, and live components.
+
+Oh, we'll also ship an Absolute utility for positioning stuff on a page.
+
+#### Alert &amp; Toast Components (Design Preview)
+We're going to share designs and documentation for our proposal on unified Alert and Toast components. As always, these will be pure UI, and not concerned with any notification engines or product specific code.
+
+### Community &amp; Education Updates
+
+#### Document how to use tokens for composition
+Guidelines on how we use Paste Tokens to build a component will be published in the Doc Site. This will enable the _last ditch effort_ for teams that need truly custom experiences but still want to use the Design System for composition.
+
+#### Brown Bag Session + Video Record on Abstract
+We'll document questions from the Abstract brown bag session that was hosted by our team, along with the video!
+
+### Process Updates
+
+#### Theme Switcher for Paste
+We'll polish the theme switching experience for Paste. This means that our implementation of SendGrid and Console themes will be more thorough. 
+
+After this is released, if you see visual or functional issues in the doc site, let us know! It's a sign that we need to fix an underlying Paste component.
+
+</Box>
+<Box marginTop={60}>
+
+## Release 0.4
+
+<Callout>
+  <CalloutText>Codenamed <strong>that one after Oh! Point Three!</strong> This release is expected to land in the <em>third week of November</em>. This release is *not locked in*, but here are candidates for the release.</CalloutText>
+</Callout>
+
+- Polish and Document Truncate Component
```

Comment:
The capitalization is all off in this list 🤔

Reply:
Changing to sentence case for this list.